### PR TITLE
Substitution to new Symmetry BC 

### PR DESCRIPTION
**Pull request by shkodm**
Adding modifications, as discussed in #61

### CODE REVIEW

File: src/d3q27/Dynamics.R [hunk 1]
```diff
@@ -28,5 +28,5 @@ AddGlobal(name="Flux", comment='Volume flux', unit="m3/s")
 
 AddNodeType("Smagorinsky", "LES")
 AddNodeType("Stab", "ENTROPIC")
-AddNodeType("SymmetryY", "BOUNDARY")
-AddNodeType("SymmetryZ", "BOUNDARY")
+AddNodeType("SymmetryYTop", "BOUNDARY")
+AddNodeType("SymmetryZIn", "BOUNDARY")
```
Comment (mdzik) [inline src/d3q27/Dynamics.R]: Change SymmetryXXX to N/S/W/E/I/OSymmetry -> like EPressure  (?)

**Comment by mdzik** [comment]
Does it work with 2D models?

**Comment by mdzik** [comment]
:+1:

**Comment by shkodm** [comment]
It's idea is similar to Bounceback function. It works, if  `Density$group`  is specified when declaring `f` declaration. For 2D, 3D, whatever. So it works fine for the d2q9_kuper for instance, but not for d2q9.

**Pull request merged**